### PR TITLE
docs(security): init

### PR DESCRIPTION
**Pull request by validcube**
<p align="center">
  <picture>
    <source
      width="256px"
      media="(prefers-color-scheme: dark)"
      srcset="assets/revanced-headline/revanced-headline-vertical-dark.svg"
    >
    
  </picture>
  <br>
  <a href="https://revanced.app/">
       <picture>
           <source height="24px" media="(prefers-color-scheme: dark)" srcset="assets/revanced-logo/revanced-logo-round.svg" />
           
       </picture>
   </a>&nbsp;&nbsp;&nbsp;
   <a href="https://github.com/ReVanced">
       <picture>
           <source height="24px" media="(prefers-color-scheme: dark)" srcset="https://i.ibb.co/dMMmCrW/Git-Hub-Mark.png" />
           
       </picture>
   </a>&nbsp;&nbsp;&nbsp;
   <a href="http://revanced.app/discord">
       <picture>
           <source height="24px" media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/13122796/178032563-d4e084b7-244e-4358-af50-26bde6dd4996.png" />
           
       </picture>
   </a>&nbsp;&nbsp;&nbsp;
   <a href="https://reddit.com/r/revancedapp">
       <picture>
           <source height="24px" media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/13122796/178032351-9d9d5619-8ef7-470a-9eec-2744ece54553.png" />
           
       </picture>
   </a>&nbsp;&nbsp;&nbsp;
   <a href="https://t.me/app_revanced">
      <picture>
         <source height="24px" media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/13122796/178032213-faf25ab8-0bc3-4a94-a730-b524c96df124.png" />
         
      </picture>
   </a>&nbsp;&nbsp;&nbsp;
   <a href="https://x.com/revancedapp">
      <picture>
         <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/93124920/270180600-7c1b38bf-889b-4d68-bd5e-b9d86f91421a.png">
         
      </picture>
   </a>&nbsp;&nbsp;&nbsp;
   <a href="https://www.youtube.com/@ReVanced">
      <picture>
         <source height="24px" media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/13122796/178032714-c51c7492-0666-44ac-99c2-f003a695ab50.png" />
         
     </picture>
   </a>
   <br>
   <br>
   Continuing the legacy of Vanced
</p>

# 🔒 Security Policy

This document describes how to report security vulnerabilities for ReVanced Manager.

## 🚨 Reporting a Vulnerability

Please open an issue in our [advisory tracker](https://github.com/ReVanced/revanced-manager/security/advisories/new) or reach out privately to us on [Discord](https://discord.gg/revanced).

If a vulnerability is confirmed and accepted, you can join our [Discord](https://discord.gg/revanced) server to receive a special contributor role.

### ⏳ Supported Versions

| Version | Branch      | Supported          |
| ------- | ------------|------------------- |
| v1.18.0 | main        | :white_check_mark: |
| latest  | dev         | :white_check_mark: |
| latest  | compose-dev | :white_check_mark: |

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      width="256px"
+      src="assets/revanced-headline/revanced-headline-vertical-light.svg"
+    >
+  </picture>
+  <br>
+  <a href="https://revanced.app/">
+       <picture>
+           <source height="24px" media="(prefers-color-scheme: dark)" srcset="assets/revanced-logo/revanced-logo-round.svg" />
+           <img height="24px" src="assets/revanced-logo/revanced-logo-round.svg" />
+       </picture>
```
Comment (Axelen123) [inline SECURITY.md:18]: These files are not present in the `compose-dev` branch.

**Review by Axelen123 — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (validcube) [reply SECURITY.md:18]: Done:tm:

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+| ------- | ------------|------------------- |
+| v1.18.0 | main        | :white_check_mark: |
+| latest  | dev         | :white_check_mark: |
+| latest  | compose-dev | :white_check_mark: |
+
+## 🚨 Reporting a Vulnerability
+
+Please open an issue in our [advisory tracker](https://github.com/ReVanced/revanced-manager/security/advisories/new) or reach out privately to us on [Discord](https://discord.gg/revanced).
+
+If a vulnerability is confirmed and accepted, you can join us [Discord](https://discord.gg/revanced) server to receive an special contributor role.
```
Comment (Axelen123) [inline SECURITY.md:77]:
```suggestion
If a vulnerability is confirmed and accepted, you can our [Discord](https://discord.gg/revanced) server to receive a special contributor role.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+     </picture>
+   </a>
+   <br>
+   <br>
+   Continuing the legacy of Vanced
+</p>
+
+# 🔒 Security Policy
+
+This document describes how to report vulnerability to ReVanced Manager.
```
Comment (Axelen123) [inline SECURITY.md:63]:
```suggestion
This document describes how to report security vulnerabilities for ReVanced Manager.
```

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+## ⏳ Supported Versions
+
+| Version | Branch      | Supported          |
+| ------- | ------------|------------------- |
+| v1.18.0 | main        | :white_check_mark: |
+| latest  | dev         | :white_check_mark: |
+| latest  | compose-dev | :white_check_mark: |
```
Comment (Axelen123) [inline SECURITY.md:71]: Maybe this should be below the "Reporting a Vulnerability" section

**Review by Axelen123 — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 2 above)
Comment (validcube) [reply SECURITY.md:77]: > ```suggestion
> If a vulnerability is confirmed and accepted, you can our [Discord](https://discord.gg/revanced) server to receive a special contributor role.
> ```

Do you mean "you can **join** our..."

**Pull request merged**